RubberduckTests/Mocks/Windows.cs

```diff
@@ -110,8 +106,8 @@ public IWindow CreateWindow(string name)
             return result.Object;
         }
 
-        public void ReleaseDockableHosts()
-        {
-        }
+        public void ReleaseDockableHosts() { }
```

Comment:
Should we clean this up? Seems silly to have a no-op method....

Reply:
That's on the `IWindows` interface - it's only a nop here because this is a test implementation.